docs/readme/MYSQL_SETUP.md

```diff
@@ -0,0 +1,35 @@
+# Setup local MySQL with Docker
```

Comment:
why `readme`? this name doesn't give any useful information. Maybe `setup`?

Reply:
updated the dir name